generators/entity-server/index.js

```diff
@@ -43,6 +43,10 @@ module.exports = class extends BaseGenerator {
         } else {
             useBlueprint = false;
         }
+
+        // Expose 'asEntity' and 'asDto' functions for templating
+        this.asEntity = name => name + this.entitySuffix;
```

Comment:
Please move these methods to generator-base.js

Reply:
Ok. I wasn't sure if the generator-base just for those functions was the right place but if you tell so, I'm 100% ok with that.

Reply:
Yes its the right place

Reply:
Just to let you know...

If I move `asEntity` and `asDto` to generator-base, I still need to define these inside a constructor (which does not yet exists in generator-base). Otherwise, the methods can't be used inside the templates.

I can do something like this : 

```javascript
module.exports = class extends PrivateBase {
  constructor(args, opts) {
        super(args, opts);
        
        // Expose asEntity and asDto to templates
        this['asEntity'] = this.asEntity;
        this['asDto'] = this.asDto;
    }

    /**
     * Construct the entity name by appending the entity suffix.
     * @param {String} name entity name
     */
    asEntity(name) {
        return name + this.entitySuffix;
    }

    /**
     * Construct the entity's dto name by appending the dto suffix.
     * @param {String} name entity name
     */
    asDto(name) {
        return name + this.dtoSuffix;
    }
};
```

But the constructor feel a little hacky... :-(

Reply:
OK I got it !
Everything is fine until a template call include(). For example :
```
<%- include('../../common/save_template', {viaService: viaService, returnDirectly: true}); -%>
```
When the function are defined inside the constructor (see above), I don't need to pass any additional values to the include function.

To fix this, I need to send more parameters to include, like this : 
```
<%- include('../../common/save_template', {asEntity; asEntity, asDto: asDto, viaService: viaService, returnDirectly: true}); -%>
```

Go with that solution ?

Reply:
hmmm, thats weird. I don't see how `this.asEntity = this['asEntity'];` is different from just declaring a class member method. Technically its the same. Anyway i'm ok with passing these to includes if needed